api/src/com/cloud/network/Network.java

```diff
@@ -215,6 +216,8 @@ public static Provider getProvider(String providerName) {
         public static final Capability DistributedRouter = new Capability("DistributedRouter");
         public static final Capability StretchedL2Subnet = new Capability("StretchedL2Subnet");
         public static final Capability RegionLevelVpc = new Capability("RegionLevelVpc");
+        public static final Capability NoVlan = new Capability("NoVlan");
+        public static final Capability PublicAccess = new Capability("PublicAccess");
```

Comment:
Can you check if an untagged capability can be used, and default egress policy can be used; instead of introducing these two capabilities?